doc/appendix-de-kassensichv/data-structures/data-structures.md

```diff
@@ -1,19 +1,19 @@
 ## Data Structures
 
-This chapter expands on the descriptions of the data structures covered in the Chapter ["Data Structures"](../../general/data-structures/data-structures.md) of the General Part, with country specific information applicable to the German market.
+This chapter expands on the descriptions of the country-specific covered in the Chapter ["Data Structures"](../../general/data-structures/data-structures.md) of the General Part, with country specific information applicable to the German market.
```

Comment:
of the country-specific Data Structures, which are covered in the respective chapter, ["Data Structures"](../../general/data-structures/data-structures.md), of the General...

Reply:
oh yes, so true, thank you!